runner/src/server/plugins/engine/pages/home.ts

```diff
@@ -1,20 +1,12 @@
 import Page from "./page";
+import { HapiRequest, HapiResponseToolkit } from "../../../types";
```

Comment:
no need to action now, but we should alias/remap so we don't have to do `../../../xxx` 🤓